File: Level 6kyu/Multiples of 3 or 5.js
function solution(number) {
    let sum = 0;
    for (let i = 0; i < number; i++) {
        if (i % 3 && i % 5) {
            continue;
        } else {
            sum += i
        }
    }
    return number < 0 ? 0 : sum;
}

File: Level 6kyu/Multiples of 3 or 5.rs
// Submitted
fn solution(num: i32) -> i32 {
    let mut result = 0;
    for n in 3..num {
        if n % 3 == 0 || n % 5 == 0 {
            result += n;
        }
    }
    return result;
}

// Run to test
fn main() {
    let res = solution(23);
    println!("{}", res);
}
